src/load/azext_load/data_plane/utils/utils.py

```diff
@@ -324,22 +326,22 @@ def convert_yaml_to_test(data):
             except InvalidArgumentValueError as e:
                 logger.error("Invalid failure criteria: %s", str(e))
             new_body["passFailCriteria"]["passFailMetrics"][metric_id] = {}
-            new_body["passFailCriteria"]["passFailMetrics"][metric_id][
-                "aggregate"
-            ] = components.split("(")[0].strip()
+            new_body["passFailCriteria"]["passFailMetrics"][metric_id]["aggregate"] = (
+                components.split("(")[0].strip()
+            )
             new_body["passFailCriteria"]["passFailMetrics"][metric_id][
                 "clientMetric"
             ] = (components.split("(")[1].split(")")[0].strip())
-            new_body["passFailCriteria"]["passFailMetrics"][metric_id][
-                "condition"
-            ] = components.split(")")[1].strip()[0]
-            new_body["passFailCriteria"]["passFailMetrics"][metric_id][
-                "value"
-            ] = components.split(
-                new_body["passFailCriteria"]["passFailMetrics"][metric_id]["condition"]
-            )[
-                1
-            ].strip()
+            new_body["passFailCriteria"]["passFailMetrics"][metric_id]["condition"] = (
+                components.split(")")[1].strip()[0]
+            )
+            new_body["passFailCriteria"]["passFailMetrics"][metric_id]["value"] = (
+                components.split(
+                    new_body["passFailCriteria"]["passFailMetrics"][metric_id][
+                        "condition"
```

Comment:
nit: move to single line `new_body["passFailCriteria"]["passFailMetrics"][metric_id]["condition"]` ?

Reply:
fixed